plugins/pencilblue/templates/angular/admin/content/objects/object_form.html

```diff
@@ -1,251 +1,210 @@
-                    wysiwygContent.push({name: fields[index].name, content: $scope.customObject[fields[index].name]});
-
-                    if(index >= fields.length - 1) {
-                        cb(wysiwygContent);
-                        return;
-                    }
-
-                    index++;
-                    self.loadWysiwyg(index);
-                    return;
-                }
-
-                var wysId = $('#' + fieldName + ' .wysiwyg').attr('id').substring('wysiwg_'.length + 1);
-                getWYSIWYGLayout(wysId, function(layout) {
-                    wysiwygContent.push({name: fields[index].name, content: layout});
-
-                    if(index >= fields.length - 1) {
-                        cb(wysiwygContent);
-                        return;
-                    }
-
-                    index++;
-                    self.loadWysiwyg(index);
-                });
-            };
-
-            this.loadWysiwyg(0);
-        };
-
-        $scope.setupWysiwygFields = function(wysiwygFields) {
-            if(wysiwygFields) {
-                $scope.wysiwygFields = wysiwygFields
-            }
-
-            for(var i = 0; i < $scope.wysiwygFields.length; i++) {
-                if($($scope.wysiwygFields[i].id).position()) {
-                    $($scope.wysiwygFields[i].id).html($scope.wysiwygFields[i].content);
-                    $scope.wysiwygFields.splice(i, 1);
-                    i--;
-                }
-            }
-
-            if($scope.wysiwygFields.length) {
-                $timeout($scope.setupWysiwygFields, 100);
-            }
-        };
-
-        for(var i = 0; i < $scope.objectType.fields.length; i++) {
-            if($scope.objectType.fields[i].field_type === 'boolean' && typeof $scope.customObject[$scope.objectType.fields[i].name] === 'undefined') {
-                $scope.customObject[$scope.objectType.fields[i].name] = true;
-            }
-            else if($scope.objectType.fields[i].field_type === 'date') {
-                $scope.customObject[$scope.objectType.fields[i].name] = $filter('date')($scope.customObject[$scope.objectType.fields[i].name], 'MM-dd-yyyy HH:mm');
-            }
-            else if($scope.objectType.fields[i].field_type === 'peer_object') {
-                $scope.customObject[$scope.objectType.fields[i].name] = $scope.customObject[$scope.objectType.fields[i].name] ? $scope.customObject[$scope.objectType.fields[i].name]._id.toString() : '';
-            }
-            else if($scope.objectType.fields[i].field_type === 'child_objects') {
-                $scope.objectType.fields[i].searchText = '';
-                $scope.objectType.fields[i].paginationIndex = 0;
-                $scope.objectType.fields[i].paginationLimit = 12;
-                $scope.paginate($scope.objectType.fields[i], 0);
-            }
+        index++;
+        self.loadWysiwyg(index);
+      });
+    };
+
+    this.loadWysiwyg(0);
```

Comment:
This pattern forces us to load one wysiwyg at a time.  We need to change this to do it in parallel to keep page rendering times down.